Source/MediaInfo/Audio/File_Aac_GeneralAudio.h

```diff
@@ -2589,22 +2589,22 @@ static const hcb_struct huffman_11[374]=
 };
 
 //---------------------------------------------------------------------------
-const int8u hcb_2step_Bytes[]=
+static const int8u hcb_2step_Bytes[]=

```

Comment:
Arne't tables duplicated in that case? I think I prefer to move the tables in a .cpp and let the .h only for table names, without static.